tcf_website/api/serializers.py

```diff
@@ -6,6 +6,13 @@
 
 class SemesterSerializer(serializers.ModelSerializer):
     """DRF Serializer for Semester"""
+    season = serializers.SerializerMethodField()
+
+    def get_season(self, obj):
+        """Change the `season` field to TitleCase (or PascalCase)"""
+        # pylint: disable=no-self-use

```

Comment:
One small thing — I think most of the other pylint disables in the codebase have been on the first (few) lines of each file as opposed to where the would-be offending code is.

Reply:
Fixed in 7c18b253bc988342486c684df611cc6978c9debc!